yelp_package/dockerfiles/trusty/Dockerfile

```diff
@@ -32,6 +37,9 @@ RUN apt-get update > /dev/null && apt-get install -y --no-install-recommends \
         libssl-dev \
         libyaml-dev \
         pyflakes \
+        python3.6 \
+        python3.6-dev \
+        python3-dev \
```

Comment:
you shouldn't need both `python3.6-dev` and `python3-dev`? are you also using python3.4 for some reason?